zerver/lib/bugdown/__init__.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+import subprocess
```

Comment:
I thought the preferred structure for dependency imports was:

1. Modules in stdlib
2. Third-party

Each one in alphabetical order. Since `subprocess` is from the standard library, that's why I put it there.

Am I wrong?